api/envoy/type/http/v3/path_transformation.proto

```diff
@@ -50,6 +57,8 @@ message PathTransformation {
       // Enable merging adjacent slashes.
       MergeSlashes merge_slashes = 3;
     }
+
+    NormalizePathAction normalize_path_action = 4;
```

Comment:
This should go into the `NormalizePathRFC3986` and `MergeSlashes` messages. For instance I may configure that NormalizePathRFC3986 has action CONTINUE but MergeSlashes is REDIRECT.

Reply:
I misunderstood which message this belongs to. Please disregard.